### PR TITLE
Added missing Reto_02_en challenge

### PR DESCRIPTION
**Pull request by its-ChaTTy**
## Summary

Created Reto02_en & added E2E tests for the same based on Issue #288 .

## Checklist

 - [x] Variables, functions and comments are translated to Spanish
 - [x] Functions follow underscore notation
 - [x] Spell check done & typos fixed
 - [x] All python code is PEP8 compliant
 - [x] Test coverage with Playwright implemented; locators are Pyhton code
 - [x] Reviewers assigned (all peers & at least 1 mentor)

### CODE REVIEW

File: _sources/challenges/Reto02_en.rst [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+
+            ====
+            from unittest.gui import TestCaseGui
+
+
+            class myTests(TestCaseGui):
+                def testOne(self):
+
+                    self.assertEqual(prep_B, 76, "Expected: Testing that prep_B has the correct value")
```
Comment (sujith-rek) [inline _sources/challenges/Reto02_en.rst:145]: write appropriate error message

File: _sources/challenges/Reto02_en.rst [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+                    self.assertEqual(list_B, B, "Testing that list_B has the correct value")
```
Comment (sujith-rek) [inline _sources/challenges/Reto02_en.rst:229]: write proper error message

**Comment by its-ChaTTy** [comment]
@sujith-rek the error messages have been updated.

**Review by NicolasSandoval — APPROVED** [review]
@its-ChaTTy Thanks. Please remove the build_info file.

**Comment by its-ChaTTy** [comment]
> @its-ChaTTy Thanks. Please remove the build_info file.

Hi @NicolasSandoval, I don't think so there is any such file in recent commit

**Comment by NicolasSandoval** [comment]
@its-ChaTTy you're right, the last commit removed it. Thanks.

**Pull request merged**